recipes/message-ix/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set version = "1.0.0-prerelease" %}
```

Comment:
So this version isn't valid. Interestingly this is noted in the CI logs, but doesn't fail the CI. Could you please send a follow-up PR to correct this? Thanks in advance.

ref: https://conda.io/docs/user-guide/tasks/build-packages/define-metadata.html#package-version

cc @isuruf

Reply:
Note to others, this is currently blocking conversion at `staged-recipes`.

cc @conda-forge/staged-recipes